packages/kbn-generate-csv/src/generate_csv.ts

```diff
@@ -327,7 +358,6 @@ export class CsvGenerator {
     let first = true;
     let currentRecord = -1;
     let totalRecords: number | undefined;
-    let totalRelation = 'eq';
```

Comment:
This line hid a bug in that `eq` would always appear in the log message even when the response JSON didn't contain a value `total.relation` field